src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/templates/bail/helper/BailDecisionUnsignedTemplateHelper.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.hmcts.reform.iacasedocumentsapi.domain.templates.bail.helper;
+
+import static uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities.BailCaseFieldDefinition.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities.BailCase;
+import uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities.ccd.CaseDetails;
+import uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities.ccd.field.YesOrNo;
+import uk.gov.hmcts.reform.iacasedocumentsapi.infrastructure.CustomerServicesProvider;
+
+@Component
+public class BailDecisionUnsignedTemplateHelper {
+
+    private final CustomerServicesProvider customerServicesProvider;
+    private final String govCallChargesUrl;
+
+    public BailDecisionUnsignedTemplateHelper(@Value("${govCallChargesUrl}") String govCallChargesUrl,
+                                              CustomerServicesProvider customerServicesProvider) {
+        this.govCallChargesUrl = govCallChargesUrl;
+        this.customerServicesProvider = customerServicesProvider;
+    }
+
+    public Map<String, Object> getCommonMapFieldValues(
+            CaseDetails<BailCase> caseDetails
+    ) {
+        final BailCase bailCase = caseDetails.getCaseData();
+
+        final Map<String, Object> fieldValues = new HashMap<>();
+        boolean isLegalRep = false;
+        final boolean hasLegalRep = bailCase.read(HAS_LEGAL_REP, YesOrNo.class).orElse(YesOrNo.NO).equals(YesOrNo.YES);
+
+        fieldValues.put("hmcts", "[userImage:hmcts.png]");
+        fieldValues.put("applicantGivenNames", bailCase.read(APPLICANT_GIVEN_NAMES, String.class));
+        fieldValues.put("applicantFamilyName", bailCase.read(APPLICANT_FAMILY_NAME, String.class));
+
+        if (bailCase.read(IS_ADMIN, YesOrNo.class).orElse(YesOrNo.NO) == YesOrNo.YES) {
+            isLegalRep = (bailCase.read(SENT_BY_CHECKLIST, String.class).orElse("").equalsIgnoreCase("legal representative"));
+        } else if (bailCase.read(IS_LEGAL_REP, YesOrNo.class).orElse(YesOrNo.NO) == YesOrNo.YES) {
+            isLegalRep = true;
+        }
+
+        final boolean isLegallyRepresentedCase = isLegalRep || hasLegalRep;
```

Comment:
there is ccd field: isLegallyRepresentedForFlag, which can be directly read to see if the case is Legally Represented. Just for future references.